### PR TITLE
Release/1.2.16

### PR DESCRIPTION
**Pull request by ikbalkaya**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.2.16](https://github.com/ably/ably-flutter/tree/v1.2.16)
+
+[Full Changelog](https://github.com/ably/ably-flutter/compare/v1.2.15...v1.2.16)
+
+**Fixed bugs:**
+
+- authHeaders type conversion on Android Plugin is incorrect [\#444](https://github.com/ably/ably-flutter/issues/444)
```
Comment (owenpearson) [inline CHANGELOG.md:9]: We should mention here the new available auth methods

**Review by owenpearson — APPROVED** [review]
Looks good, one suggestion though:

File: CHANGELOG.md (hunk 1 above)
Comment (ikbalkaya) [reply CHANGELOG.md:9]: 9e7b3a8531075050a9ef08b4f0f9c6f3ecaf7850

Comment (ikbalkaya) [reply CHANGELOG.md:13]: 5a0c17c36d48a0fc1a44c392136dba2d64c6f6cd

**Review by owenpearson — APPROVED** [review]
LGTM, thanks!

**Pull request merged**